tests/foreman/api/test_convert2rhel.py

```diff
@@ -195,6 +195,25 @@ def oracle(
             '/etc/firewalld/firewalld.conf && firewall-cmd --reload'
         )
         assert result.status == 0
+    assert oracle_host.execute('yum -y update').status == 0
+
+    if major == '8':
+        # Set RHEL kernel to be used during boot
+        oracle_host.execute("mkdir -p /boot/loader/entries/backup")
```

Comment:
Since we've one more rememdiation above yum update step for OEL8, can we incorporate that as well in this block?